src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,105 @@
+package duke.command;
```

Comment:
I like the way you have segmented your code in separate command classes in the command package! It's very organised and readable.